index.html

```diff
@@ -69,56 +77,60 @@
-        </div>
-        <div class="image-blurb">
-          <img src="http://placehold.it/140x100"><h3>Curriculum Info 6</h3><p>Info blub</p>  
+        <div class="curriculum-blurbs">
+          <div class="image-blurb">
+            <img src="http://placehold.it/140x100"><h3>Curriculum Info 1</h3><p>Info blub</p>
+          </div>
+           <div class="image-blurb">
+            <img src="http://placehold.it/140x100"><h3>Curriculum Info 2</h3><p>Info blub</p>
+          </div>
+          <div class="image-blurb">
+            <img src="http://placehold.it/140x100"><h3>Curriculum Info 3</h3><p>Info blub</p>
+          </div>
+          <div class="image-blurb">
+            <img src="http://placehold.it/140x100"><h3>Curriculum Info 4</h3><p>Info blub</p>
+          </div>
+          <div class="image-blurb">
+            <img src="http://placehold.it/140x100"><h3>Curriculum Info 5</h3><p>Info blub</p>
+          </div>
+          <div class="image-blurb">
+            <img src="http://placehold.it/140x100"><h3>Curriculum Info 6</h3><p>Info blub</p>  
+          </div>
         </div>
       </section>
 
       <section class="application">
         <h2>APPLICATION PROCESS</h2>
-        <div class="application-steps">
-          <ol>
-            <li>
-              <h3>Check your eligibility</h3>
-              <p>Do you meet the following criteria</p>
-            </li>
-            <li>
-              <h3>Fill out application form</h3>
-              <p>The application helps us get to know you better. Answer questions about your interest in technology and what gets you excited about the program.</p>
-            </li>
-            <li>
-              <h3>Take entrace assessment</h3>
-              <p>Learn the fundamentals of networking and take this online entrace assessment to prove your knowledge.</p>
-            </li>
-          </ol>  
-        </div>
-        <div class="eligibility">
-          <h3>Eligibility Requirements</h3>
-          <ul>
-            <li>at least 18 years old</li>
-            <li>less than $50K salary</li>
-            <li>less than a 4-year degree</li>
-            <li>NYC resident</li>
-            <li>meet all federal selective service requirements</li>
-          </ul>
+        <div class="curriculum-blurbs">
+          <div class="application-steps">
+            <ol>
+              <li>
+                <h3>Check your eligibility</h3>
+                <p>Do you meet the following criteria</p>
+              </li>
+              <li>
+                <h3>Fill out application form</h3>
+                <p>The application helps us get to know you better. Answer questions about your interest in technology and what gets you excited about the program.</p>
+              </li>
+              <li>
+                <h3>Take entrace assessment</h3>
+                <p>Learn the fundamentals of networking and take this online entrace assessment to prove your knowledge.</p>
+              </li>
```

Comment:
Let's add an application step:

Interview
Selected candidates will be invited to interview for a spot in the cohort.